lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -94,15 +77,19 @@ export abstract class ClientApplication {
     }
 
     protected buildOauthClientConfiguration(): Configuration {
+        // using null assertion operator as we ensure that all config values have default values in buildConfiguration()
         return {
             authOptions: this.config.auth,
```

Comment:
Forgot one?

Reply:
Don't need one here as we are not accessing any of the properties in `this.config.auth`. 

If we we accessing`this.config.auth.clientId` for example, we would then have to assert that `this.config.auth` is not null by using `this.config.auth!.clientId`.